cd/before-deploy.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
-    openssl aes-256-cbc -K $encrypted_SOME_key -iv $encrypted_SOME_iv -in cd/signingkey.asc.enc -out cd/signingkey.asc -d
-    gpg --fast-import cd/signingkey.asc
+    openssl aes-256-cbc -K $encrypted_5b0511f84952_key -iv $encrypted_5b0511f84952_iv -in codesigning.asc.enc -out cd/codesigning.asc -d
```

Comment:
these variables correspond to what? $encrypted_5b0511f84952_key $encrypted_5b0511f84952_iv

Why there is a kind of id:hash in it?

Reply:
see comment above

Reply:
this is a generated number when doing this operation: http://www.debonair.io/post/maven-cd/#encrypt-cert:a547b4a31e9ae1ba41fe3873843c9208